app/models/data_request.rb

```diff
@@ -35,7 +38,9 @@ class DataRequest < ApplicationRecord
     pending: 1,
     processing: 2,
     failed: 3,
-    complete: 4
+    complete: 4,
+    canceling: 11,
```

Comment:
Minor not blocking: Why the jump to 11, 12 instead of incrementing by 1?

Reply:
it seemed like a good idea to separate the new status codes from the old ones because they were for a different purpose, but it's not terribly important either way